src/bower/DependenciesManager.js

```diff
@@ -77,10 +77,41 @@ define(function (require, exports) {
         return FileUtils.exists(path);
     }
 
+    /**
+     * Get the dir of the dependencies
+     * @param {string=} path
+     * @return {Promise}
+     */
+    function findBowerJsonLocation(path) {
+        if (!path) {
+            path = ProjectManager.getProjectRoot().fullPath;
+        }
+
+        return path;
+    }
+
     function _loadBowerJson(path) {
         _bowerJsonFile = new BowerJsonFile(path);
     }
 
+    function getDependencies() {
+        var deferred = new $.Deferred();
+
+        _bowerJsonFile.getContent()
+            .done(function (data) {
+                var dependencies = [];
+                data = JSON.parse(data);
+
+                _.each( data.dependencies, function (version, name) {
```

Comment:
I prefer to use Array.prototype.forEach, and it is also following the brackets code conventions.